### ISSUE TITLE
Bundle type of 'image' is not supported despite javapackager command line accepting it

### ISSUE THREAD

**Issue by swpalmer** [FibreFoX/javafx-gradle-plugin#92]
When I set
jfx {
bundler = 'image'
}

I get:

* What went wrong:
Execution failed for task ':MyProject:jfxNative'.
> No bundler found for given name image. Please check your configuration.

However help output of javapackager indicates:
  -native <type>
          generate self-contained application bundles (if possible).
          If type is specified then only bundle of this type is created.
          List of supported types includes: installer, image, exe, msi, dmg, rpm, deb.

and sure enough 'image' is accepted on the command line

**Comment by FibreFoX** [comment]
Yes, this was intentionally. In previous versions of the JDK 8 there seemed to be no further use of that filtering-type, and I personally find it better to have tighter control about the generated files than "just anything possible".

Does this really feel missing?

**Comment by swpalmer** [comment]
I think so.  "image" doesn't mean "just anything possible".  That's what not specifying any bundler means.  "image" however, creates the application bundle with the java launcher, .cfg files, app folder, runtime folder, etc., and then I can incorporate those files into a custom installer.

**Comment by FibreFoX** [comment]
There were four values as an enum:
```
NONE, IMAGE, INSTALLER, ALL
```
The problem with this is, that you have no real control about what bundler will get used. This is kind of a bundler-group-name, where `NONE` and `ALL` had special meaning.
[Looking at the JDK 10](https://docs.oracle.com/javase/10/tools/javapackager.htm#GUID-E51F9601-E121-4A50-BCA7-C7F8730078B2__GUID-659E9890-5F31-479B-94CA-58B374DB9A28) this still seems to be existing, but the whole concept feels broken for me.

Will re-evaluate this the next days.
